templates/routing-informs/README.md

```diff
@@ -0,0 +1,38 @@
+# Nextmv INFORMS Analytics 2023 routing template
+
+The INFORMS template sets up a vehicle routing problem (VRP) solver that is
+compatible with our [cloud](https://docs.nextmv.io/cloud/get-started) input
+files. In addition to almost being a drop-in replacement for the cloud endpoint
+it also demonstrates some of the more advanced router options.
+
+The most important files created are `main.go` and several data input files for
+different uses cases: fleet management, delivery, distribution and sourcing. By
```

Comment:
Can we remove the note about different use cases? That’s I think copy paste error from cloud routing template.

Reply:
Also I think we need some edits to this since you brought the code into a single file